test/unit/io/stream/ostream_test_template.hpp

```diff
@@ -46,15 +44,17 @@ TYPED_TEST_P(ostream, output)
     std::ifstream fi{filename.get_path(), std::ios::binary};
     std::string buffer{std::istreambuf_iterator<char>{fi}, std::istreambuf_iterator<char>{}};
 
-    if constexpr (std::same_as<TypeParam, contrib::gz_ostream> || std::same_as<TypeParam, contrib::bgzf_ostream>)
+    if constexpr (std::same_as<TypeParam,
+                               seqan3::contrib::gz_ostream> || std::same_as<TypeParam,
+                               seqan3::contrib::bgzf_ostream>)
```

Comment:
Here please align the `std::same_as` and leave their template args on the same line.